test/unit/bulk/common.test.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+const { expect } = require('chai');
+const { mergeBatchResults } = require('../../../src/bulk/common');
+const { Timestamp, Long } = require('../../../src/bson');
+
+describe('bulk/common', function () {
+  describe.only('#mergeBatchResults', function () {
```

Comment:
```suggestion
  describe('#mergeBatchResults', function () {
```
just noticed the stray .only